beetmoverscript/utils.py

```diff
@@ -161,6 +171,26 @@ def generate_beetmover_template_args(context):
     return tmpl_args
 
 
+def _generate_beetmover_template_args_maven(task, release_props):
+    tmpl_args = {
+        'artifact_id': task['payload']['artifact_id'],
+        'branch': release_props['branch'],
+        'buildid': release_props['buildid'],
+        'product': release_props['appName'],
+        'template_key': 'maven_{}'.format(release_props['appName']),
+        'version': task['payload']['version'],
+    }
+
+    if 'nightly' in tmpl_args['artifact_id']:
```

Comment:
I'm confused. Can you **not** have `nightly` there as a string? I'm looking at https://tools.taskcluster.net/groups/TJW4b-CNSwGsJukgQCeoUA/tasks/PojKrc74Q5KFqghgPlosCg/details 🤔 

... later edit: never mind me. I found https://github.com/mozilla-releng/beetmoverscript/pull/163/files#diff-8a0baf3b191b3039ec72b8da4e543908R490

Reply:
Yeah, nightly occurs on maple and mozilla-central. `artifact_id` is gotten from _MOZ_UPDATE_CHANNEL_PER_BRANCH in https://phabricator.services.mozilla.com/D2421 which maps what the build system creates. 

For the record, I didn't manage to get the config file from the build system in-tree. I just found a mozconfig on maple: https://searchfox.org/mozilla-central/rev/033d45ca70ff32acf04286244644d19308c359d5/testing/mozharness/configs/single_locale/maple_android-api-16.py#28

Reply:
Hm, I feel you, would've been better if it came from in-tree I suppose. This might be fragile in the future but sounds like it works for now; The only place I know that var is as well is [this](https://hg.mozilla.org/mozilla-central/file/tip/browser/confvars.sh) but I don't think that helps much.